src/declaration.d

```diff
@@ -996,6 +996,8 @@ public:
     Expression edtor;               // if !=null, does the destruction of the variable
     IntRange* range;                // if !=null, the variable is known to be within the range
 
+    ubyte _isDataSeg;               // 0b00 unset, 0b01 true, 0b11 false
```

Comment:
The binary literals are rather strange. I just use 0, 1 or 2. Using enum is a bit of overkill. Just change the comment to something like "private data for isDataSeg". Also, no leading _. Just call it `isdataseg`.